frameworklauncher/doc/USERMANUAL.md

```diff
@@ -587,7 +587,7 @@ Launcher sets up below EnvironmentVariables for each User Service to use:
 | EnvironmentVariable | Description |
 |:---- |:---- |
 | CONTAINER_GPUS | Only valid when gpuNumber is greater than 0. It is a number, each bit of this number represents a Gpu, for example, 3 represents gpu0 and gpu1. |
-| CONTAINER_PORTS | Only valid when portDefinitions is not empty. It is a string, a format example is: "portLabel1:port1,port2,port3;portLabel2:port4,port5,port6". |
+| CONTAINER_PORTS | Only valid when portDefinitions is not empty. It is a string, a format example is: "portLabel1:port1,port2;portLabel2:port4,port5;". |
```

Comment:
>; [](start = 126, length = 1)

where is port3 ?  1,2, 4, 5 ?

Reply:
OK

---
In reply to: [189219910](https://github.com/Microsoft/pai/pull/588#discussion_r189219910) [](ancestors = 189219910)